db/docker-compose.yml

```diff
@@ -1,13 +1,18 @@
+# Set the version of docker compose to use
 version: '3.6'
+
+# The containers that compose the project
 services:
-    postgres:
+    db:
         image: postgres:13
         restart: always
         volumes:
             - db_data:/var/lib/postgresql/data
         ports:
-            - '${POSTGRES_PORT}:5432'
+            - '3003:5432'
```

Comment:
Is this from another commit/rebase or intentional?

Reply:
It was intentional, I was getting an error when trying to run it like that. Not sure why it was working before but won't work now. You're welcome to take a look and see if you can get it working with that, but I don't think it's necessary since the port is the same for both scripts.

Reply:
When you say it doesn't work, do you mean locally or in CI?

Reply:
Locally, but even if it worked on ci if it doesn't work locally that's a problem